hack/release/packaging/utils/lib.sh

```diff
@@ -1,3 +1,5 @@
+#!/bin/bash
+
```

Comment:
The purpose for adding the shebang line is to be able to parse the file correctly with `shellcheck`. I haven't done a full linting pass on these files, but I intend to do so in the future, so that's why this line was added. If you think it's a better idea to leave it unchanged though, I'll revert this.

Reply:
I don't want to impose my view here, but I do think it's wrong for tools to drive an incorrect agenda.  Perhaps there's another way to tell shellcheck that this is a file that it should check?